crates/prover/src/components/memory.rs

```diff
@@ -19,10 +17,14 @@ use stwo_prover::core::pcs::TreeVec;
 use stwo_prover::core::poly::circle::{CanonicCoset, CircleEvaluation};
 use stwo_prover::core::poly::BitReversedOrder;
 
+use crate::adapter::memory::MemoryBoundaries;
 use crate::relations;
 
-#[derive(Copy, Clone, Default)]
+const N_M31_IN_MEMORY_ENTRY: usize = 7;
+
+#[derive(Clone, Default)]
 pub struct Claim {
+    pub inputs: MemoryBoundaries,
```

Comment:
The `Claim` is included in the proof, but we don't want to include the whole bounds in the public memory (so far)